src/info_page.cpp

```diff
@@ -170,8 +170,10 @@ QPair<bool, QString> InfoPage::translateInfoParams(QString key)
 	} else if (key.contains("ip")) {
 		key = "IP Address";
 	} else if (key.startsWith("cal,gain") ||
-		   key.startsWith("cal,offset")) {
-		key = "";
+		   key.startsWith("cal,offset") ||
+		   key.startsWith("cal,temp_lut")) {
+//		key = "";
```

Comment:
nitpick: if this is not needed it should be removed

Reply:
fixed